docker/domserver/Dockerfile

```diff
@@ -10,12 +10,8 @@ RUN apt-get update \
 	gcc g++ make acl zip unzip \
 	php-cli php-zip \
 	php-gd php-curl php-mysql php-json php-intl \
-	php-gmp php-xml php-mbstring \
```

Comment:
I think xml is/was used for the icpc cms integration but might be wrong

Reply:
That could explain why I didn't need it when rebuilding the image.